Sources/MySQLNIO/MySQLConnection.swift

```diff
@@ -3,7 +3,7 @@ import Logging
 import NIOSSL
 import NIOPosix
 
-public final class MySQLConnection: MySQLDatabase {
+public final class MySQLConnection: MySQLDatabase, @unchecked Sendable {
```

Comment:
Are we guaranteeing that shared state is safe? Is there a reason this has to be unchecked?